src/main/java/com/github/onsdigital/dp/image/api/client/exception/ImageAPIException.java

```diff
@@ -0,0 +1,20 @@
+package com.github.onsdigital.dp.image.api.client.exception;
+
+/**
+ *  ImageAPI Exception
+ */
+public class ImageAPIException extends Exception {
+    private int code;
+
+    public ImageAPIException(String message, int code) {
```

Comment:
Java doc - `message` is obvious but specifying what `code` is will be useful to anyone using this class.